README.md

```diff
@@ -75,6 +76,9 @@ require'nvim-web-devicons'.setup {
  -- prevents cases when file doesn't have any extension but still gets some icon
  -- because its name happened to match some extension (default to false)
  strict = true;
+ -- set the light or dark variant manually, instead of relying on `background`
+ -- (default to "auto")
```

Comment:
`"auto"` is not really needed and will be a little confusing.

Let's remove it.

Reply:
Changed to nil.